ZenPacks/zenoss/Import4/bin/imp4_run.sh

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env bash
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2014, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+# Parse the arguments
+
+DESIRED_USER=${DESIRED_USER:-"zenoss"}
+# temporary line to copy /import4 from development code to the zenpack egg image
+cmd="cp -rp /import4/pkg/* /opt/zenoss/ZenPacks/ZenPacks.zenoss.Import4-0.0.9dev-py2.7.egg/ZenPacks/zenoss/Import4"
```

Comment:
Yes, this dev code will be removed after we exit the development stage.. It should be in next iteration. 

It is here to copy the latest dev code out of a shared area into the egg tree.